test/integration/routes/siti-agri/agreements.test.js

```diff
@@ -0,0 +1,32 @@
+import { retrieveOrganisationAgreements } from '../../../../src/factories/siti-agri/agreement.factory.js'
+
+describe('Basic queries for faked routes', () => {
+  let mockServer
+  const PROCESS_ENV = process.env
+
+  beforeAll(async () => {
+    process.env = { ...PROCESS_ENV }
+    process.env.PORT = '3097' // Set to obscure port to avoid conflicts
+    const { startServer } = await import('../../../../src/server.js')
+    mockServer = await startServer()
+  })
+  afterAll(() => {
+    process.env = PROCESS_ENV
+    mockServer.stop({ timeout: 0 })
+  })
+
+  describe('SitiAgri route', () => {
```

Comment:
It's probably also useful to check the returned data matches the schema:
```suggestion
  describe('SitiAgri route', () => {
    it('should GET a agreements conforming to the schema', async () => {
      const { result, statusCode } = await server.inject({
        method: 'GET',
        url: '/SitiAgriApi/cv/agreementsByBusiness/sbi/1111111111/list'
      })
      expect(statusCode).toBe(200)
      expect(result).toConformToSchema(
        schema.paths['/SitiAgriApi/cv/agreementsByBusiness/sbi/{sbi}/list'].get.responses['200'].schema
      )
    })

```